src/app/tasks/components/TaskLoggerItem/TaskLoggerItem.tsx

```diff
@@ -181,6 +188,7 @@ const TaskLoggerItem = ({ notification, task }: TaskLoggerItemProps): JSX.Elemen
           isHovered={isHovered}
           status={notification.status}
           progress={progress.toString()}
+          nItems={nItems.toString()}
```

Comment:
I have seen that `nItems` can be undefined, this would give error in that case. Is it checked?